app/models/broadcast.rb

```diff
@@ -32,4 +34,12 @@ def single_active_announcement_broadcast
 
     errors.add(:base, "You can only have one active announcement broadcast")
   end
+
+  def update_last_active_at
+    # Updates the last_active_at timestamp to show
+    # when the Broadcast was last set to "active"
```

Comment:
IMHO, I don't think this comment is adding anything; the code is pretty self-documenting here :)